ci-operator/jobs/redhat-operator-ecosystem/release/redhat-operator-ecosystem-release-master-presubmits.yaml

```diff
@@ -0,0 +1,175 @@
+presubmits:
+  openshift/release:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/ci-operator-config
+    decorate: true
+    name: pull-ci-redhat-operator-ecosystem-release-master-ci-operator-config
+    rerun_command: /test ci-operator-config
+    spec:
+      containers:
+      - args:
+        - --config-dir
+        - ./ci-operator/config/
+        command:
+        - ci-operator-checkconfig
+        image: ci-operator-checkconfig:latest
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          requests:
+            cpu: 10m
+    trigger: (?m)^/test( | .* )ci-operator-config,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/ci-operator-registry
+    decorate: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-redhat-operator-ecosystem-release-master-ci-operator-registry
+    rerun_command: /test ci-operator-registry
+    spec:
+      containers:
+      - args:
+        - --config
+        - ./ci-operator/config/
+        - --registry
+        - ./ci-operator/step-registry/
+        - --validate-only
+        command:
+        - ci-operator-configresolver
+        image: ci-operator-configresolver:latest
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          requests:
+            cpu: 10m
+    trigger: (?m)^/test( | .* )ci-operator-registry,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/generated-config
+    decorate: true
+    name: pull-ci-redhat-operator-ecosystem-release-master-generated-config
+    rerun_command: /test generated-config
+    spec:
+      containers:
+      - command:
+        - hack/validate-generated-config.sh
+        image: ci-operator-prowgen:latest
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          requests:
+            cpu: 10m
+    trigger: ((?m)^/test generated-config,?(\s+|$))
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/ordered-prow-config
+    decorate: true
+    name: pull-ci-redhat-operator-ecosystem-release-master-ordered-prow-config
+    rerun_command: /test ordered-prow-config
+    spec:
+      containers:
+      - command:
+        - hack/validate-prow-job-ordering.sh
```

Comment:
On all of these that use specific scripts, we could keep one source of truth by doing:

```yaml
extra_refs:
- base_ref: master
  org: openshift
  repo: release
# ...
  spec:
    containers:
    - command:
      - ./../../openshift/release/hack/validate-prow-job-ordering.sh
```

Reply:
Changed to single source of truth model